### DISCUSSION TITLE
Internationalization

### DISCUSSION THREAD

**Discussion by sweetmantech** [rainbow-me/rainbowkit#406] [Ideas]
Can we add support for other languages?

I'm living in Argentina. Most of the people here speak Spanish. They want to understand Web3, but there aren't many UIs supporting Spanish.

RainbowKit has the potential to be the wallet provider of the developing world.

**Comment by DanielSinclair** [comment]
@SweetmanTech This is now available with [v1.1.0](https://github.com/rainbow-me/rainbowkit/releases/tag/%40rainbow-me%2Frainbowkit%401.1.0). `es-419` for Latin American Spanish is available today; let us know if you spot any low quality translations and we can improve them. This will work automatically based on a user's preferred browser language, but you can reference our docs for best practices and implementation details for more advanced approaches: https://www.rainbowkit.com/docs/localization
